Project-Descriptions-and-Plans/CVA6/CVA6-main-plan-approved-2024+.md

```diff
@@ -0,0 +1,526 @@
+    - Thales India / Engineering Competence Center (ECC)
+- The University of Minho
+- Zero-Day Labs
+- PULP team (ETH Zürich and U. Bologna)
+- 10xEngineers
+- MU-Electronics
+- CEA
+- PlanV
+- Axelera
+- A partner of the TRISTAN project
+
+## Project Structure
+
+The CVA6 project has evolved to a main project and several sub-projects as more CVA6 configurations are envisioned to target TRL5.
+
+### Common goals
+
+Despite the number of sub-projects, the following goals will be pursued between the main and sub-projects:
+- Optimize the effort sharing and reuse between configuraitons
+- Keep a common RTL database (with parameters)
+- Still be able to accept relevant evolutions (but be more selective than in the past)
+- Manage a common documentation
+
+The scopes of the main, sub-, and sister projects are described below:
+
+### Main project
+
+The main CVA6 project includes:
+- The synchronization with the sub-projects
+- The review and selection of proposed CVA6 evolutions<sup>2</sup>
+- The review of pull requests (PRs) from sister projects and the external world
+- The 2<sup>nd</sup> level review of PRs from sub-projects: check that the PR integrates well in CVA6 repository
+- The maintenance of the GitHub-triggered CI flow, the CVA6 GitHub repository , the triage of external issue
+- Design activities that are not hosted in a sub- or sister project (MMU unification, FPGA optimizations...)
+- The definition of a release process, that is needed by sub-projects.
+
+Note:<br>
+<sup>2</sup> A sister project and/or a TWG gate can be recommended for major evolutions.
+
+### Sub-projects
+
+Each CVA6 configuration targetting the TRL5 maturity will launch a sub-project with a PA gate presented at a TWG meeting.
+
+The sub-projects mostly include verification activities.
+They also host design and documentation activities that are specific to the configuration.
+
+The PA gates of sub-projects include:
+- Quality target (coverage targets, certification if any...)
+- Additions to the RTL freeze checklist as deemed necessary
+- Identification of reused/joint/new verification building blocks (DV plans, agents, sequences)
+- Doc/Design/DVPlan/sequences creation/updates in synchronization with other configurations
+- 1<sup>st</sup> level review of PRs from the sub-project: check that the PR meets the sub-project goals and quality targets
+- Fix issues discovered in the sub-project
+- Release plan for the configuration, based on the release process
+
+The goal of the 2<sup>nd</sup> level review is to verify that the PR integrates well in CVA6 common repository.
+
+The sub-project PA gates are prepared and presented by their respective leaders.
+Prior to presenting the gate to the TWG, it is recommended to review the sub-project and PA gate document in a CVA6 meeting.
+
+At the time of writing, the envisioned sub-projects are:
+- CV32A60AX<sup>3</sup>, a 32-bit application core, led 10xEngineers
+- CV32A60X<sup>4</sup>, a 32-bit embedded core, led by TSS
```

Comment:
The performance/power/gate count trade-off is very good for the dual issue version. That is why TSS targets only 65x, no more 60x.

Reply:
I'll change the wording accordingly.